openbb_platform/providers/fmp/openbb_fmp/models/etf_equity_exposure.py

```diff
@@ -0,0 +1,93 @@
+"""FMP ETF Equity Exposure Model."""
+
+# pylint: disable=unused-argument
+
+import asyncio
+import warnings
+from typing import Any, Dict, List, Optional
+
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.etf_equity_exposure import (
+    EtfEquityExposureData,
+    EtfEquityExposureQueryParams,
+)
+from openbb_core.provider.utils.errors import EmptyDataError
+from openbb_core.provider.utils.helpers import amake_request
+from pydantic import field_validator
+
+_warn = warnings.warn
+
+
+class FMPEtfEquityExposureQueryParams(EtfEquityExposureQueryParams):
+    """
+    FMP ETF Equity Exposure Query Params.
+
+    Source: https://site.financialmodelingprep.com/developer/docs/etf-stock-exposure-api/
+    """
+
+
+class FMPEtfEquityExposureData(EtfEquityExposureData):
+    """FMP ETF Equity Exposure Data."""
+
+    __alias_dict__ = {
+        "equity_symbol": "assetExposure",
+        "etf_symbol": "etfSymbol",
+        "shares": "sharesNumber",
+        "weight": "weightPercentage",
+        "market_value": "marketValue",
+    }
+
+    @field_validator("weight", mode="before", check_fields=False)
+    @classmethod
+    def normalize_percent(cls, v):
+        """Normalize percent values."""
+        return float(v) / 100 if v else None
+
+
+class FMPEtfEquityExposureFetcher(
+    Fetcher[FMPEtfEquityExposureQueryParams, List[FMPEtfEquityExposureData]]
+):
+    """FMP ETF Equity Exposure Fetcher."""
+
```

Comment:
I think somewhere in here we now allow for : ```    __json_schema_extra__ = {"symbol": ["multiple_items_allowed"]} ``` to indicate that multiple symbols are allowed?

Reply:
Unclear on this, @IgorWounds, where does this go?

Reply:
This goes under the QueryParams of a provider-specific model. Check out any `price.historical` model